backend/apid/actions/namespaces.go

```diff
@@ -75,15 +75,15 @@ func (a OrganizationsController) Create(ctx context.Context, newOrg types.Organi
 	}
 
 	// Persist
-	if err := a.Store.CreateOrganization(ctx, &newOrg); err != nil {
+	if err := a.Store.CreateNamespace(ctx, &newOrg); err != nil {
```

Comment:
Perhaps we should update the variable name here to avoid confusion with the removed `organization` field.
```suggestion
	if err := a.Store.CreateNamespace(ctx, &newNamespace); err != nil {
```